pkg/api/defaults-kubelet.go

```diff
@@ -157,6 +162,11 @@ func (cs *ContainerService) setKubeletConfig() {
 			}
 		}
 
+		// For Ubuntu 18.04 VMs
+		if profile.IsUbuntu1804() {
+			profile.KubernetesConfig.KubeletConfig["--resolv-conf"] = "/run/systemd/resolve/resolv.conf"
```

Comment:
This is to fix container networking. Many thanks to @cpuguy83 for helping me come up with the workaround.

https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/1624320/comments/8